test/application/management/AppDirectory.test.js

```diff
@@ -0,0 +1,114 @@
+const AppDirectory = artifacts.require('AppDirectory')
+const assertRevert = require('../../helpers/assertRevert')
+const ContractDirectory = artifacts.require('ContractDirectory')
+const shouldBehaveLikeContractDirectory = require('../versioning/ContractDirectory.behavior')
+
+contract('ContractDirectory', ([_, owner, fallbackOwner, anotherAddress, implementation_v0, implementation_v1, fallbackImplementation]) => {
```

Comment:
The contract name is `AppDirectory`.
Also, consider renaming `fallbackImplementation -> stdImplementation`